src/main/java/com/google/devtools/build/lib/rules/java/JavaOptions.java

```diff
@@ -71,41 +71,34 @@ public ImportDepsCheckingLevelConverter() {
               + " instead; see #7849")
   public boolean disallowLegacyJavaToolchainFlags;
 
-  private static final String DEFAULT_JAVABASE = "@bazel_tools//tools/jdk:jdk";
-
   @Deprecated
   @Option(
       name = "javabase",
```

Comment:
Do you want to move these to [`BazelGraveyardOptions`](https://cs.opensource.google/bazel/bazel/+/master:src/main/java/com/google/devtools/build/lib/bazel/rules/BazelRulesModule.java;bpv=1;bpt=1;l=53?ss=bazel&q=graveyard&gsn=BuildGraveyardOptions&gs=kythe%3A%2F%2Fgithub.com%2Fbazelbuild%2Fbazel%3Flang%3Djava%3Fpath%3Dcom.google.devtools.build.lib.bazel.rules.BazelRulesModule.BuildGraveyardOptions%23d805dab9a7881823fba3226a83075552e8e9a4e3c1b9e28d48de5691b32db6cc)?

Reply:
I rather keep them here. We had some issues when an option was not copied to all of the *GraveyardOption.

Reply:
SGTM